cclient/request/get_trytes.c

```diff
@@ -20,17 +20,15 @@ void get_trytes_req_free(get_trytes_req_t** const req) {
     return;
   }
 
-  get_trytes_req_t* tmp = *req;
-
-  if (tmp->hashes) {
-    flex_hash_array_free(tmp->hashes);
+  if ((*req)->hashes) {
+    hash243_queue_free(&(*req)->hashes);
   }
-  free(tmp);
+
+  free(*req);
   *req = NULL;
 }
 
-get_trytes_req_t* get_trytes_req_add_hash(get_trytes_req_t* const req,
-                                          tryte_t const* const hash) {
-  req->hashes = flex_hash_array_append(req->hashes, hash);
-  return req;
+retcode_t get_trytes_req_add_hash(get_trytes_req_t* const req,
+                                  flex_trit_t const* const hash) {
```

Comment:
Why are some `_*add` functions taking trytes and some flex_trits ?

Reply:
the main idea is the request/response should take flex_trit_t, but `tyrtes` is provided for the convenience in test cases at json_serializer and ciri api. I think this function should be removed or changed it to take `tyrtes` for consistency.